<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RulesSetting</class>
 <widget class="QWidget" name="RulesSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1454</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RulesSetting</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>681</width>
     <height>661</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableWidget" name="tableWidget_Rule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>256</width>
      <height>541</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>102</y>
      <width>373</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="Modify_Button">
       <property name="text">
        <string>修改</string>
       </property>
       <property name="btnStyle" stdset="0">
        <string>normalBtnpink</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Add_Button">
       <property name="text">
        <string>追加</string>
       </property>
       <property name="btnStyle" stdset="0">
        <string>normalBtnpink</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>灰度值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QTableWidget" name="tableWidget_Rules">
       <column>
        <property name="text">
         <string>No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>灰度值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>显示符号</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditGrayValue"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>显示字符</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditGrayWord"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Delete_Button">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="btnStyle" stdset="0">
        <string>normalBtnpink</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>规则设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>——————————————————————————————————————————————————————————</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
